src/plugins/core/borders.ts

```diff
@@ -152,6 +164,29 @@ export class BordersPlugin extends CorePlugin<BordersPluginState> implements Bor
     return border;
   }
 
+  /**
+   * Return the colors of all borders of a sheet
+   * @param sheetId Id of the sheet that we want to get the borders
+   * @returns All the colors used in the borders of the sheets
```

Comment:
this js doc is mostly useless IMHO. The method signature and name are self-explanatory